journals/apps/journals/journal_page_helper.py

```diff
@@ -5,10 +5,32 @@
 from django.core.cache import cache
 from django.shortcuts import redirect
 from django.utils.cache import add_never_cache_headers
+from wagtail.wagtailcore.models import PagePermissionTester, UserPagePermissionsProxy
 
 from journals.apps.journals.utils import get_cache_key
 
 
+class JournalPagePermissionTester(PagePermissionTester):
+    def can_delete(self):
+        """
+            if page is one of from non_deletable_pages_from_cms Page models, returns False
+            otherwise super model logic will work.
+        :return:
+        """
+        from journals.apps.journals.models import JournalIndexPage, JournalAboutPage
+        non_deletable_pages_from_cms = (JournalAboutPage, JournalIndexPage)
```

Comment:
nit: we can avoid single use variable `non_deletable_pages_from_cms` here and update the statement below like this
`if isinstance(self.page.specific, (JournalAboutPage, JournalIndexPage)):`